libvirt/tests/cfg/backingchain/blockcopy/blockcopy_with_xml_option_to_luks_slice_image.cfg

```
@@ -0,0 +1,33 @@
+- backingchain.blockcopy.xml_option.luks_slice_image:
+    type = blockcopy_with_xml_option_to_luks_slice_image
+    basic_option = " --transient-job --reuse-external --verbose --wait"
+    private_key_password = "EXAMPLE_PWD"
+    secret_pwd = "`printf %s ${private_key_password} | base64`"
+    file_dest_extra = "--object secret,data=${private_key_password},id=sec0 -o encrypt.format=luks,encrypt.key-secret=sec0,preallocation=full "
+    file_dest_format = "qcow2"
+    src_disk_format = "qcow2"
+    file_dest_path = "/var/lib/libvirt/images/file_dest_path.qcow2"
+    slice_offset = 0
+    sec_dict = {"secret_ephemeral": "no", "secret_private": "yes", "description": "LUKS Secret", "usage": "volume", "volume": "/var/lib/libvirt/images/luks-sample.img"}
+    src_disk_path = "/var/lib/libvirt/images/source_test.qcow2"
+    image_size = "500M"
+    target_disk = "vdb"
+    xml_path = " --xml {} "
+    variants:
+        - finish_reuse_external:
+            operation = "finish"
+            blockcopy_options = " --finish ${basic_option}"
+        - pivot_reuse_external:
+            operation = "pivot"
+            blockcopy_options = " --pivot ${basic_option}"
+    variants:
+        - file_disk:
+            source_disk_type = "file"
+            dest_disk_type = "file"
+            disk_dict = {"type_name":"${source_disk_type}", "target":{"dev": "${target_disk}", "bus": "virtio"}, "driver": {"name": "qemu", "type":"qcow2"}}
+        - block_disk:
+            blk_dest_format = "raw"
+            source_disk_type = "file"
+            dest_disk_type = "block"
+            disk_dict = {"type_name":"file", "target":{"dev": "${target_disk}", "bus": "virtio"}, "driver": {"name": "qemu", "type":"qcow2"}}
```

How about updating as below?
```suggestion
            disk_dict = {"type_name":"${source_disk_type}", "target":{"dev": "${target_disk}", "bus": "virtio"}, "driver": {"name": "qemu", "type":"qcow2"}}
```

---

done